### DISCUSSION TITLE
Action not triggered when a release was published

### DISCUSSION THREAD

**Discussion by sstone** [community/community#36448] [Actions]
Hello,

We have set up a github action worfklow to be executed whenever we publish a new release through github's UI (create new release draft then publish). It has worked will so far.

The trigger is set with:
```
on:
  release:
    types: [published]
```

We went through the same steps today (the only difference being that we had already created the release tag instead of letting github create it upon publication, and that we did not tag the latest commit on our master branch but instead a branch that has not been merged), but publishing our release did not trigger our github action. Does anyone know why and how we can "fix it" i.e. get this action to run ?

**Comment by rabelenda** [comment]
I just experienced this same issue, and after deleting the release, and associated tag, and recreating the release, the action was triggered 🤷‍♂️

**Reply by rabelenda** [reply]
I added, as a workaround for the future, to the release workflow a `workflow_despatch` event trigger, in case I experience this same thing in the future to manually trigger it.

**Reply by sstone** [reply]
Thanks! I'll try this

**Comment by ossanna16** [comment]
Hey there @sstone and thank you for asking a good question! I'm glad you found our community 🙂
